CHANGELOG_PENDING.md

```diff
@@ -2,6 +2,18 @@
 
 ## v0.34.21
 
+Release highlights include:
+- A new RPC configuration flag, `discard_abci_responses`, which, if enabled,
+  discards all ABCI responses except the latest one in order to reduce disk
+  space usage in the state store. Note that, if enabled, this means that the
+  `block_results` RPC endpoint will only be able to return the latest block.
+- A CLI command, `reindex-event`, is introduced to re-index block and tx events
+  to the event sinks. You can run this command when the event store backend
+  dropped/disconnected or you want to replace the backend. Note that, if
+  `discard_abci_responses` is enabled, you will not be able to use this command.
+
+See below for more details on specific changes in this release.
```

Comment:
this is probably fine in this context, but "relative references" like "see below" in english end up assuming things that aren't always true about the rendering of the page, and can be confusing. 

It's probably fine to just omit this entirely.